webapp/app.py

```diff
@@ -592,13 +594,16 @@ def takeovers_index():
 
 @app.after_request
 def cache_headers(response):
+
     """
-    Set cache expiry to 60 seconds for homepage and blog page
+    Set cache to private for image builder and advantage to prevent caching
     """
-
     if flask.request.path in ["/core/build", "/advantage"]:
         response.cache_control.private = True
 
+    """
+    Set cache expiry to 60 seconds for homepage and blog page
+    """
```

Comment:
Docstrings are only for the top of functions. If you want an inline comment it looks like this:

```suggestion
    # Set cache expiry to 60 seconds for homepage and blog page
```